src/sentry/options/defaults.py

```diff
@@ -553,6 +553,9 @@
 register("sentry-metrics.cardinality-limiter.limits.releasehealth.per-org", default=[])
 register("sentry-metrics.cardinality-limiter.orgs-rollout-rate", default=0.0)
 
+# Flag to determine whether abnormal_mechanism tag should be extracted
+register("sentry-metrics.releasehealth.abnormal-mechanism-extraction-rate", default=0.0)
```

Comment:
does this belong in `options/defaults.py` or `projectoptions/defaults.py`?

Reply:
This location is correct, as this is a global option and project options are tied to projects.